src/Raven.Client/Documents/Replication/ReplicationNode.cs

```diff
@@ -13,7 +13,7 @@ namespace Raven.Client.Documents.Replication
     /// <summary>
     /// Data class for replication destination
     /// </summary>
-    public abstract class ReplicationNode : IEquatable<ReplicationNode>
+    public class ReplicationNode : IEquatable<ReplicationNode>, IDynamicJson
```

Comment:
the fix is adding `IDynamicJson` ?

Reply:
removing the `abstract` class. I added the `IDynamicJson` interface to guarantee the implementation of `ToJson`